repo/content/index.go

```diff
@@ -19,7 +19,7 @@ type packIndex interface {
 
 	ApproximateCount() int
 
-	GetInfo(contentID ID) (Info, error)
+	GetInfo(contentID ID) (Info, error) // may return ErrContentNotFound
```

Comment:
Q: what's the motivation for returning `ErrContentNotFound`? Is it due to the "nil nil" linter? just curiosity, not saying it should not be done.

This has or may have a slight change in semantics, and would be a good candidate to have put in a separate PR by itself to make it easier to reason through it.

Reply:
I'm going to suppress the linter and move this commit out to separate PR.